server/services/interventionsService.test.ts

```diff
@@ -2910,6 +2911,59 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
   */
+
+  describe('getServiceProviderReportingData', () => {
+    const reportingResponse = [
+      {
+        referralLink: 'https://refer-and-monitor.example/referral/c9f9e22f-ddd9-422a-a9af-76fc02b18b38',
+        referralRef: 'SM1973AC',
+        referralId: 'c9f9e22f-ddd9-422a-a9af-76fc02b18b38',
+        contractId: 'thing',
```

Comment:
May just be dummy data but this should be a uuid right?

Reply:
I don't think I can answer that one. In the v1.0.0 schema (which I linked to in another comment here) it says "Format of string TBC". There is however a [comment from @sldblog](https://dsdmoj.atlassian.net/browse/IC-1116?focusedCommentId=161187) saying that this perhaps should be called `contract_reference`, since the UUID will have no meaning to the suppliers. I'm not sure if it's too late to make those changes now, though.